src/event.js

```diff
@@ -504,6 +565,77 @@ jQuery.event = {
 	}
 };
 
+// Ensure the presence of an event listener that handles manually-triggered
+// synthetic events by interrupting progress until reinvoked in response to
+// *native* events that it fires directly, ensuring that state changes have
+// already occurred before other listeners are invoked.
+function leverageNative( el, type, forceAdd, allowAsync ) {
+
+	// Setup must go through jQuery.event.add
+	if ( forceAdd ) {
+		jQuery.event.add( el, type, forceAdd );
+		return;
+	}
+
+	// Register the controller as a special namespace-universal handler
+	dataPriv.set( el, type, forceAdd );
+	jQuery.event.add( el, type, {
+		namespace: false,
+		handler: function( event ) {
+			var maybeAsync, result,
+				saved = dataPriv.get( this, type );
+
+			// Interrupt processing of the outermost synthetic .trigger()ed event
+			if ( ( event.isTrigger & 1 ) && this[ type ] && !saved ) {
+
+				// Store arguments for use when handling the inner native event
+				saved = slice.call( arguments );
+				dataPriv.set( this, type, saved );
+
+				// Trigger the native event and capture its result
+				// Support: IE <=9 - 11+
```

Comment:
This suggests it may still happen in some future version (or its de-facto evolution, i.e. the EdgeHTML-based Edge) but here we know that's not the case. I usually try to end the 11-closed IE comment with `only` instead:
```suggestion
				// Support: IE <=9 - 11 only
```

Reply:
I prefer the 11+ form to indicate that the issue was not actually resolved within IE, and would presumably still exist if the browser is revived. But I'm willing to defer to @jquery/core consensus here.

Reply:
I guess it mattered more when EdgeHTML was still in development and `11+` might suggest to some that newer engine is still affected. This will soon no longer be the case once a few Chromium-based Edge releases come out and we stop testing on EdgeHTML.

I'd still want us to be consistent here but I'll be doing a support comments cleanup when we start working on 4.0 anyway so it's not necessary to settle on something now. Hence, I'm resolving the discussion.